process/block/preprocess/transactions.go

```diff
@@ -480,6 +479,9 @@ func (txs *transactions) CreateAndProcessMiniBlock(
 		}
 
 		if addedGasLimitPerCrossShardMiniblock+currTxGasLimit > process.MaxGasLimitPerMiniBlock {
```

Comment:
Can we extract this in a variable or a function returning a boolean with a descriptive name?

Reply:
Done